### PR TITLE
XW-4282 | Removing unnecessary tracking

### PR DESCRIPTION
**Pull request by rybmat**
## Links
https://wikia-inc.atlassian.net/browse/XW-4282

## Description

Removing unnecessary tracking

## Reviewers

@Wikia/x-wing

### CODE REVIEW

**Review by latata — APPROVED** [review]

File: app/controllers/application.js [hunk 1]
```diff
@@ -58,23 +55,7 @@ export default Controller.extend(
 				language: this.get('wikiVariables.language')
 			});
 
-			// This event is for tracking mobile sessions between Mercury and WikiaMobile
-			// NOTE: this event won't have additional dimensions set up from API, ie. #19 (articleType)
-			track({
```
Comment (latata) [inline app/controllers/application.js:63]: Why did you remove it?

Comment (rybmat) [reply app/controllers/application.js:63]: @hakubo said it is not needed anymore

**Review by hakubo — APPROVED** [review]
